Md 857

protitude opened the pull request:
**Ticket:**
https://cyberteamportal.atlassian.net/jira/software/c/projects/D8/issues/D8-857?filter=myopenissues

**URLS:**
https://md-857-accessmatch.pantheonsite.io/add-skill
https://md-857-accessmatch.pantheonsite.io/add-interest

a-pasquale commented:
I wonder about moving the JavaScript to the Footer of the view. Maybe wouldn't need the setTimeout?

protitude commented:
Good call on the JS in the footer. I moved it there and removed the delay. I disabled all views below the 3rd level and enabled caching on the skills view and switched to tag based caching on the interest view. Hopefully that will help a bit.
